Task_2_2_1/src/main/java/r/nsu/fit/masud/Warehouse.java

```diff
@@ -0,0 +1,54 @@
+package r.nsu.fit.masud;
+
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class Warehouse {
```

Comment:
Your warehouse should provide safe methods for different thread could work with it

See Java synchronized, wait, notify, notifyAll